google/stackdriver_monitoring/metric_collector.py

```diff
@@ -17,23 +17,81 @@
 
 # pylint: disable=missing-docstring
 import argparse
+import collections
 import logging
 import logging.config
-import collections
+import socket
+import threading
+import time
 
 from http_server import (HttpServer, StdoutRequestHandler)
 
 import metric_collector_handlers as handlers
 import stackdriver_handlers as stackdriver_handlers
 
-from spectator_client import SpectatorClient
-from stackdriver_client import StackdriverClient
+import spectator_client
+import stackdriver_client
 
 
 HandlerDefinition = collections.namedtuple(
   'HandlersDefinition', ['handler', 'url_path', 'command_name', 'description'])
 
 
+class DummyMetricService(object):
+  def store(self, service_metrics):
+    spectator_client.foreach_metric_in_service_map(service_metrics, self.dump)
+
+  def dump(self, service, name, metadata, instance):
+    name, tags = spectator_client.normalize_name_and_tags(
+        name, metadata, instance)
+    if tags is None:
+      print 'IGNORE {0}.{1}'.format(service, name)
```

Comment:
Is this formatting (and some in the last PR) in a format required by stackdriver?